apps/www/src/lib/components/docs/copy-button.svelte

```diff
@@ -2,12 +2,52 @@
 	import { clickToCopyAction } from "svelte-legos";
 	import { cn } from "$lib/utils";
 	import { Check, Copy } from "radix-icons-svelte";
+	import { Button } from "@/registry/default/ui/button";
+	import * as DropdownMenu from "@/registry/default/ui/dropdown-menu";
 
 	let copied = false;
+	let commands: Record<"npm" | "yarn" | "pnpm" | "bun", string> = {
+		npm: "",
+		yarn: "",
+		pnpm: "",
+		bun: ""
+	};
 	let className: string | undefined | null = undefined;
 	export let value = "";
 	export { className as class };
 
+	$: if (value) {
+		// npm install
+		if (value.startsWith("npm install")) {
+			commands = {
+				npm: value,
+				yarn: value.replace("npm install", "yarn add"),
+				pnpm: value.replace("npm install", "pnpm add"),
+				bun: value.replace("npm install", "bun add")
+			};
+		}
+
+		// npx create
+		else if (value.startsWith("npx create-")) {
+			commands = {
+				npm: value,
+				yarn: value.replace("npx create-", "yarn create "),
+				pnpm: value.replace("npx create-", "pnpm create "),
+				bun: value.replace("npx create-", "bunx --bun")
```

Comment:
```suggestion
				yarn: value.replace("npx create ", "yarn create "),
				pnpm: value.replace("npx create ", "pnpm create "),
				bun: value.replace("npx create ", "bunx --bun")
```

I'm not seeing anywhere that a `-` follows create, so I think this is causing the `npm create svelte@latest my-app` block to not work like the others!

Reply:
I honestly copied the code snippet I linked in the PR description and didn't change it much 😅 I just added the `yarn dlx` command and put `else`s between conditions and that's it!

That being said, it's a good observation! However, as both can apparently exist (see [npm docs](https://docs.npmjs.com/cli/v10/commands/npm-init)), I'm in favor of merging both options!